### PR TITLE
LOOP-1850 Fix pause icon color

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-1850

### CODE REVIEW

File: DashKit/Mocks/MockPodCommManager.swift [hunk 1]
```diff
@@ -317,33 +317,41 @@ public class MockPodCommManager: PodCommManagerProtocol {
             return
         }
         
+        // SDK returns .invalidProgram if bolus attempt is made during suspension.
```
Comment (rickpasetto) [inline DashKit/Mocks/MockPodCommManager.swift:320]: @ps2 Just wondering...how are the changes in this file related to the pause icon color?

**Review by rickpasetto — COMMENTED** [review]
Question for Pete....

File: DashKit/Mocks/MockPodCommManager.swift (hunk 1 above)
Comment (ps2) [reply DashKit/Mocks/MockPodCommManager.swift:320]: The delay is from a comment Matt made while testing this ticket, that he didn't see the expected spinner. The .invalidProgram something I added when I was testing and noticed we could bolus while suspended, which didn't seem right. So I tested the actual PodSDK, and it returns this error.

Comment (ps2) [reply DashKit/Mocks/MockPodCommManager.swift:320]: Both are just making the mock pod behave a little more like a real pod. No effects on actual shipped Loop.

**Review by rickpasetto — APPROVED** [review]
LGTM. 👍

**Pull request merged**